templates/publisher/listing.html

```diff
@@ -511,6 +518,9 @@ <h2 class="p-heading--four">Additional information</h2>
         {% for screenshot_url in screenshot_urls %}
           { url: {{ screenshot_url|tojson }}, type: "screenshot", status: "uploaded" },
         {% endfor %}
+        {% for banner_url in banner_urls %}
+          { url: {{ banner_url|tojson }}, type: "banner", status: "uploaded" },
```

Comment:
Since we assume there is only one banner (and only read one in JS anyway), could it be just filtered as one image here, and not passed as unnecessary array that we read just first item of?